src/api/actions/index.ts

```diff
@@ -1,3 +1,5 @@
 export * from "./uploadMedia";
 export * from "./uploadMetadata";
 export * from "./uploadObject";
+export * from "./checkBulkUploadJob";
+export * from "./startBulkUpload";
```

Comment:
nit that I won't block on, but run the formatter before you PR code for cleanliness / consistency (`Alt+Shift+F`)

Reply:
fixed this